boards/swervolf/README.md

```diff
@@ -0,0 +1,77 @@
+SweRVolf
+========
+
+SweRVolf is a FuseSoC-based SoC for the SweRV RISC-V core.
+
+This can be used to run the RISC-V compliance tests, Zephyr OS
+or other software in simulators or on FPGA boards. Focus is on
+portability, extendability and ease of use; to allow SweRV users
+to quickly get software running, modify the SoC to their needs or
+port it to new target devices.
+
+https://github.com/chipsalliance/Cores-SweRVolf
+
+Running
+-------
+
+FuseSoC can be used to run SweRVolf on either an FPGA board or in a simulation
+enviroment.
+
+Tock has been tested on the simulation enviroment.
+
+Running in simulation with EH1 core
+-----------------------------------
+For full details in setting up the FuseSoC simulator see: https://github.com/chipsalliance/Cores-SweRVolf#prerequisites
+
+The quick steps to setup are shown below:
+
+First install verilator
+
+```shell
+sudo pacman -S verilator
```

Comment:
Can we change this to not be Arch Linux (or any other pacman-based distribution) specific? I imagine just specifying the required dependencies and their tested version would be great.

Reply:
I would prefer to leave it as is. Tock has lots of distro/OS specific steps (like OSX and Ubuntu) so I don't think it's strange to include this. It also makes it easier for people to get started and for me to remember how to set it up :P

Reply:
Fair enough. Care to include the version for this and the other packages at least? It's not unlikely to have ancient packages (looking at you, Debian). Having a version reference (at least some lower boundary) rules out a lot of errors.

Reply:
The full SweRVolf contains that type of information. I would rather leave it there instead of duplicating it here.

Reply:
Fair enough. I think this will be the place where people are going to look, but it's up to you.